### PR TITLE
cl-stitch: optimize parameters for quality

### PR DESCRIPTION
**Pull request by liuyinhangx**
* fix rounding error for offset, so that only adjust
   stitch-area if the offset is equal or greater than 4
 * fix counting error in debug mode

### CODE REVIEW

File: modules/ocl/cv_feature_match.cpp [hunk 1]
```diff
@@ -279,15 +290,13 @@ detect_and_match (
 
     cv::Rect tmp_crop_left = crop_left;
     cv::Rect tmp_crop_right = crop_right;
-    crop_left.x -= x_offset;
+    crop_left.x -= x_offset - 1.0f;
```
Comment (windyuan) [inline modules/ocl/cv_feature_match.cpp]: maybe 0.5f is better. what do you think?

**Comment by liuyinhangx** [comment]
Update patch, split XCAM_ALIGN_AROUND into XCAM_ALIGN_DOWN and XCAM_ALIGN_UP.

**Comment by liuyinhangx** [comment]
update patch, please help to review.

**Comment by windyuan** [comment]
merged. thanks.
commit 3006239ec517139261f69e5ade52445104e88dc7

**Pull request closed without merging**